backend/database/greenlist_registry_db.sql

```diff
@@ -108,19 +108,19 @@ CREATE TABLE reclaims
 
-        ('Organic Linen', 'Another natural fabric which has similar qualities to hemp, but is made from flax, is linen. Sadly, the way that conventional linen is produced pollutes waterways and uses harmful chemicals. However, the crop does not need to be harvested this way! In certain environments flax can be grown without fertilizer.', 'http://localhost:11500/images/materials/organicLinen.jpg', 'The United Nations Food and Agriculture Organisation state that flax uses 13 times less pesticides than potatoes, but is only approximately 1% of the world’s apparel fibre consumption.The United Nations Food and Agriculture Organisation state that flax uses 13 times less pesticides than potatoes, but is only approximately 1% of the world’s apparel fibre consumption.', ' In the right conditions, linen can be cultivated without fertilizers and grown on marginal land that is unsuitable for food crop production. Linen is also one of the most biodegradable fabrics when untreated.', 'Generally, line that is produced in Europe has a much lower ecological footprint and is higher quality. While line produced in China is cheaper, is also of lower quality and involves the use of argo-chemicals and water processing.'),
+        ('Organic Linen', 'Another natural fabric which has similar qualities to hemp, but is made from flax, is linen. Sadly, the way that conventional linen is produced pollutes waterways and uses harmful chemicals. However, the crop does not need to be harvested this way! In certain environments flax can be grown without fertilizer.', 'http://localhost:11500/images/materials/organicLinen.jpg', 'The United Nations Food and Agriculture Organisation state that flax uses 13 times less pesticides than potatoes, but is only approximately 1% of the world’s apparel fibre consumption.', ' In the right conditions, linen can be cultivated without fertilizers and grown on marginal land that is unsuitable for food crop production. Linen is also one of the most biodegradable fabrics when untreated.', 'Generally, line that is produced in Europe has a much lower ecological footprint and is higher quality. While line produced in China is cheaper, is also of lower quality and involves the use of argo-chemicals and water processing.'),
```

Comment:
@KathyPuma extra space at beginning of one column, after '. also linen missing second 'n' twice